android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -529,6 +529,13 @@ public void setJavaScriptEnabled(WebView view, boolean enabled) {
     view.getSettings().setJavaScriptEnabled(enabled);
   }
 
+  @ReactProp(name = "androidHardwareAccelerationDisabled")
+  public void setHardwareAccelerationDisabled(WebView view, boolean disabled) {
+    if (disabled) {
+      view.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
+    }
```

Comment:
maybe re-enable it in an else case ?

Reply:
Thanks for your suggestion, from the guide [
Hardware acceleration](https://developer.android.com/guide/topics/graphics/hardware-accel#controlling)  currently user cannot enable hardware acceleration at the view level.

Reply:
you are 100% right